fprodents/workflows/base.py

```diff
@@ -327,10 +325,6 @@ def init_single_subject_wf(subject_id):
               ('outputnode.template', 'inputnode.template'),
               ('outputnode.anat2std_xfm', 'inputnode.anat2std_xfm'),
               ('outputnode.std2anat_xfm', 'inputnode.std2anat_xfm')]),
-            (bold_ref_wf, brain_extraction_wf, [
-                ('outputnode.epi_ref_file', 'inputnode.in_files')]),
-            (brain_extraction_wf, func_preproc_wf,
-             [("outputnode.out_mask", "inputnode.bold_mask")]),
```

Comment:
does `func_preproc_wf` not need an input mask anymore?

Reply:
No - in this case we transform the `anat_mask` from `anat_preproc_wf` after registering the `ref_file` in `func_preproc_wf`